internal/server/handler_v2.go

```diff
@@ -74,19 +75,40 @@ func fetchRemote(
 func (s *Server) V2Resolve(
 	ctx context.Context, in *pbv2.ResolveRequest,
 ) (*pbv2.ResolveResponse, error) {
-	resp, err := V2ResolveCore(ctx, s.store, s.mapsClient, in)
-	if err != nil {
-		return nil, err
-	}
-	if s.metadata.RemoteMixerDomain != "" {
-		remoteResp := &pbv2.ResolveResponse{}
-		err := fetchRemote(s.metadata, s.httpClient, "/v2/resolve", in, remoteResp)
+	errGroup, errCtx := errgroup.WithContext(ctx)
+	respChan := make(chan *pbv2.ResolveResponse, 2)
+
+	errGroup.Go(func() error {
+		resp, err := V2ResolveCore(errCtx, s.store, s.mapsClient, in)
```

Comment:
just confirm V2ResolveCore can return nil (instead of empty protobuf)

Reply:
Yes, the returned type is *pbv2.ResolveResponse, which is a pointer, so "nil" is just an empty pointer.

Reply:
Also, I think "nil" is in general preferred over a non-nil pointer to an empty protobuf.